docs/reference/schemas/tags_parameters.yaml

```
@@ -125,14 +125,15 @@ properties:
     in: query
     name: traces_tags
     description: |
-      The traces_tags of product(s) you are searching for. 
+      The traces_tags of product(s) you are searching for.
       The [OFF App](https://world.openfoodfacts.org/traces) shows a list of possible values for `traces`.
   tag_name_with_language_code:
     in: query
+    name: tag_name_with_language_code
```

I would use the one I used below:
```suggestion
    name: '<tag_name>_tags_<language_code>'
```